src/java.base/share/classes/java/text/Collator.java

```diff
@@ -226,7 +226,13 @@ public static synchronized Collator getInstance() {
     }
 
     /**
-     * Gets the Collator for the desired locale.
+     * Gets the Collator for the desired locale. If the desired locale
+     * has the "{@code ks}" and/or the "{@code kk}"
+     * <a href="https://www.unicode.org/reports/tr35/tr35-collation.html#Setting_Options">
+     * Unicode collation settings</a>, the returned {@code Collator} instance
+     * will have the specified strength and/or decomposition respectively. If the specified
+     * setting value is not recognized, strength and/or decomposition will not be
+     * overridden.
      * @apiNote Implementations of {@code Collator} class may produce
```

Comment:
The API should identify the valid values for ks and kk and the corresponding values as passed to setStrength and setDecomposition.